openage/convert/main.py

```diff
@@ -16,12 +16,18 @@
 from ..util.fslike.directory import CaseIgnoringDirectory
 from ..util.strings import format_progress
 
+_STANDARD_PATH_IN_32BIT_WINEPREFIX =\
+    "drive_c/Program Files/Microsoft Games/Age of Empires II/"
+_STANDARD_PATH_IN_64BIT_WINEPREFIX =\
+    "drive_c/Program Files/Microsoft Games (x86)/Age of Empires II/"
```

Comment:
If it's a 32bit wineprefix, there will only be `Program Files/`, and in it age.
If it's a 64bit wineprefix, there will be both directories, and age be in the `(x86)` variant.